From 393e4f565b759338531464eb65edd73f3b247724 Mon Sep 17 00:00:00 2001
From: Ile Eftimov <ileeftimov@gmail.com>
Date: Wed, 2 Sep 2015 15:41:09 +0200
Subject: [PATCH 1/1] set CALC_EXACT_MALLOC_SIZE to 1

---
 gc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gc.c b/gc.c
index b843e7b..7b88753 100644
--- a/gc.c
+++ b/gc.c
@@ -275,7 +275,7 @@ static ruby_gc_params_t gc_params = {
 #define GC_ENABLE_LAZY_SWEEP   1
 #endif
 #ifndef CALC_EXACT_MALLOC_SIZE
-#define CALC_EXACT_MALLOC_SIZE 0
+#define CALC_EXACT_MALLOC_SIZE 1
 #endif
 #if defined(HAVE_MALLOC_USABLE_SIZE) || CALC_EXACT_MALLOC_SIZE > 0
 #ifndef MALLOC_ALLOCATED_SIZE
-- 
2.2.1

